docs/Migrate-from-V1-to-V2.md

```diff
@@ -107,7 +107,7 @@ As a result, your Survey Creator component code can look as follows:
 ```javascript
 import React, { Component } from "react";
 import * as SurveyCreator from "survey-creator-react";
-import "survey-core/modern.css";
+import "survey-core/defaultV2.css";
 import "survey-creator-core/survey-creator-core.css";
 
```

Comment:
Can you make the following changes to the lines below?

Line 149:
`// Register a new property editor for the "shorttext" property type`

Please add a period at the end of the comment.

Line 166:
Can you update this line as follows?

`We do not use an "after render" technique in the SurveyJS Creator V2 to render additional actionable widgets for survey elements. We now use standard platform-dependent techniques to render them on the design surface: as templates - for Knockout, as React components - for React.`

Line 167:
Can you update this line as well?

`As a result, the old code used to render custom adorners will not work in V2, and SurveyJS Creator V2 ignores this code.`

Line 171:
Please change the period at the end to a colon.